src/conn/api_strerror.c

```diff
@@ -43,6 +43,21 @@ __wt_wiredtiger_error(int error)
         return ("WT_TRY_SALVAGE: database corruption detected");
     }
 
+    /*
+     * Check for WiredTiger specific sub-level errors.
+     */
```

Comment:
Can be formatted on one line:
```suggestion
    /* Check for WiredTiger specific sub-level errors. */
```

Reply:
Fixed in most recent commit